model/app.model.js

```diff
@@ -15,3 +15,18 @@ exports.fetchAllEndpoints = () => {
     })
 }
 
+exports.fetchArticleById = async (article_id) => {
+    const sizeArticlesQuery = `SELECT COUNT(*) FROM articles;`
```

Comment:
We don't need to make this extra query here. If the array comes back empty, we can reject the promise